### PR TITLE
In-App notifications (redux logic)

### PR DESCRIPTION
**Pull request by btzr-io**
### WIP
See: https://github.com/lbryio/lbry-desktop/issues/695

### CODE REVIEW

File: src/redux/reducers/notifications.js [hunk 1]
```diff
@@ -39,6 +63,25 @@ reducers[ACTIONS.DISMISS_NOTIFICATION] = state => {
...
+  const { index } = action.data;
+  stack.splice(index, 1);
+
+  return Object.assign({}, state, {
+    stack,
+  });
+};
+
+reducers[ACTIONS.CLEAR_NOTIFICATIONS] = state => {
+  const stack = Object.assign([], state.stack);
```
Comment (btzr-io) [inline src/redux/reducers/notifications.js]: Is this necessary?

Comment (btzr-io) [reply src/redux/reducers/notifications.js]: can it be just `const stack = [];` ?

**Comment by btzr-io** [comment]
Ready for review ^^

**Comment by seanyesmunt** [comment]
Will give this a more in-depth review, but is there any way we could add this to the existing notification stack? We would probably need additional values like the time/is_read but I think that would be preferable over keeping it separate.

**Pull request closed without merging**